tests/e2e/e2e_test.go

```diff
@@ -123,6 +54,12 @@ var _ = Describe("odoe2e", func() {
 			getProj := runCmd("odo project get --short")
 			Expect(strings.TrimSpace(getProj)).To(Equal(projName))
 		})
+
+		// Failing: Issue #630
+		// It("should list the project", func() {
+		// 	listProj := runCmd("odo project list")
```

Comment:
this test need not be commented 
replace `listProj := runCmd("odo project list")`  with `listProj := runCmd("sleep 2s && odo project list")`

refer comment on https://github.com/redhat-developer/odo/issues/630#issuecomment-416883762